demo/components/number-input.demo.tsx

```diff
@@ -21,7 +25,13 @@ export class NumberInputDemo extends React.Component<{}, {value?: number}> {
                     <span data-slot="suffix">suffix</span>
                 </NumberInput>
             </td><td>
-                <NumberInput disabled value={this.state.value} />
+                <NumberInput
+                    className="number-input"
+                    disabled
+                    value={this.state.value}>
```

Comment:
can you pls move `>` into next line so it will be possible to see where attrs ends and children starts

```jsx
<NumberInput
    className='...'
   disabled
>
    <span data-slot="prefix">prefix</span>
</NumberInput>
```